admin_manual/ai/app_context_chat.rst

```diff
@@ -0,0 +1,65 @@
+=================
+App: Context Chat
+=================
+
+.. _ai-app-context_chat:
+
+Context Chat is an :ref:`assistant<ai-app-assistant>` feature that is implemented via an ensemble of two apps:
+
+ * the *context_chat* app, written purely in PHP
+ * the *context_chat_backend* ExternalApp written in Python
+
+Together they provide the ContextChat text processing tasks accessible via the :ref:`Nextcloud Assistant app<ai-app-assistant>`.
+
+The *context_chat* and *context_chat_backend* apps run only open source models and do so entirely on-premises. Nextcloud customers receive customer support upon request.
+
+As the models are entirely open source, the quality of translations may not be comparable to commercially available machine translation services.
+
+Requirements
+------------
+
+* The *context_chat_backend* app is built as an External App and thus depends on AppAPI v2.3.0
+* Nextcloud AIO is supported
+* Using GPU processing is supported, but not required; be prepared for slow performance unless you are using GPU
```

Comment:
It is required unfortunately since the docker image is configured to only support GPU due to several constraints (like llama-cpp's custom compilation and configuration of model loading in config.yaml), and that too only NVIDIA gpus, not AMD or Intel's ARC.